kitsune/sumo/static/js/gallery.js

```diff
@@ -256,8 +265,8 @@
             self.uploadSuccess($input, iframeJSON, options.filename);
         },
         /*
-         * Fired after upload is complete, if isValidFile is true, and server
-         * returned succes.
+         * Fired after upload is complete, if isValidFile and isNotTooLarge
+         * are true, and server returned succes.
```

Comment:
Same here, `isNotTooLarge` is an implementation details of `isValidFile`, not something that needs documented on this function.